benchmark/shear_bands/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.8)
 
-FIND_PACKAGE(Aspect REQUIRED HINTS ${ASPECT_DIR} ../ ../../ $ENV{ASPECT_DIR})
+FIND_PACKAGE(Aspect REQUIRED HINTS ${Aspect_DIR} ../ ../../ $ENV{ASPECT_DIR})
```

Comment:
Isn't there a form of the kind `${value-or-empty-if-not-exist:ASPECT_DIR}`? I don't recall the exact syntax, but there should be something that allows using it even if not set.